# Use the optimization options from EasyBuild
# Davide Vanzo (Vanderbilt University)
diff -ru charmm.orig/build/UNX/Makefile_em64t charmm/build/UNX/Makefile_em64t
--- charmm.orig/build/UNX/Makefile_em64t	2019-04-16 15:08:16.873426458 -0500
+++ charmm/build/UNX/Makefile_em64t	2019-04-16 15:09:31.909428534 -0500
@@ -88,8 +88,8 @@
 
 FC0 = $(FC) -c -O0 -free -fp-model strict
 FC1 = $(FC) -c -O1 -free -fp-model strict
-FC2 = $(FC) -c -O3 -mp1 -axSSE4.1 -free -fp-model strict
-FC3 = $(FC) -c -O3 -mp1 -axSSE4.1 -free -fp-model strict
+FC2 = $(FC) -c -mp1 -free $(F90FLAGS)
+FC3 = $(FC) -c -mp1 -free $(F90FLAGS)
 FCR = $(FC) -c -u -V -free -fp-model strict
 FCD = $(FC) -c -g -O0 -u -traceback -free
 
